extensions/setup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+import sys
+
+if '..' not in sys.path:
+    sys.path.insert(0, '..')
+import setup as mypy_setup
```

Comment:
This (importing the top-level setup.py) also scares the living daylight out of me.

Most setup.py files are purely cargo-culted and work by accident.